.circleci/config.yml

```diff
@@ -459,8 +473,71 @@ jobs:
           root: ~/
           paths:
             - project/opennms-container/sentinel/images/
+  horizon-deb-build:
+    executor: debian-build-executor
+    # Larger memory footprint required to speed up builds using Takari smartbuilder
+    resource_class: large
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Build Debian Packages
+          command: |
+            export NODE_OPTIONS=--max_old_space_size=1024
+            export CCI_MAXCPU=4
+            .circleci/scripts/makedeb.sh opennms
```

Comment:
Will this end up rebuilding the `core/web-assets` project again? We had to add workaround to avoid this for the RPMs.

Reply:
I think I actually fixed that with https://github.com/OpenNMS/opennms/commit/b4bdafad6e83a2b4689047361bf0d3476c6fd481 -- I'll double-check the logs though to be sure.

Reply:
Aha!  No, it still ran more than it needed to, I added the `-PskipCompile` to the Debian version of the assembly build.